api/src/declare/declare.service.ts

```diff
@@ -0,0 +1,69 @@
+import { Injectable, Logger, BadGatewayException } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { lastValueFrom } from 'rxjs';
+import { Client } from '@hubspot/api-client';
+
+@Injectable()
+export class DeclareService {
+  private readonly logger = new Logger(DeclareService.name);
+  private readonly hubspotApiUrl = 'https://api.hubapi.com';
+
+  constructor (private readonly httpService: HttpService) {}
+  
+  
+  async getDeclarations(): Promise<{ signatures: string }> {
+    const formId = 'f51c1352-c778-40a8-b589-b911c31e64b1';
+    const hubspotToken = process.env.HUBSPOT_TOKEN;
+
+    let response;
+    try {
+      response = await lastValueFrom(
+        this.httpService.get(
+          `${this.hubspotApiUrl}/form-integrations/v1/submissions/forms/${formId}`,
+          {
+            headers: {
+              Authorization: `Bearer ${hubspotToken}`,
+            },
+          },
+        ),
+      );
+    } catch (error: any) {
+      this.logger.error(
+        `Failed to fetch data from HubSpot API: ${error.message}`,
+        error?.response?.data || error.stack,
+      );
+      throw new BadGatewayException('Failed to fetch data from Hubspot API');
+    }
+
+    const data = response.data?.results || [];
+
+    console.log('Full response:', response);
+    console.log('Data:', data);
+
+    const uniqueSignatures = new Set<string>();
+
+    if (data?.length) {
+      for (const submission of data) {
+        if (submission.values.length > 0) {
+          let firstName = submission.values[0].value;
+          let lastName = submission.values[1].value;
+          // format the names to look nice and prevent duplicates.
+          if (firstName && firstName.length >= 2) {
+            firstName =
+              firstName.charAt(0).toUpperCase() + firstName.slice(1).toLowerCase().trim();
+          }
+          if (lastName && lastName.length >= 2) {
+            lastName =
+              lastName.charAt(0).toUpperCase() + lastName.slice(1).toLowerCase().trim();
+          }
+
+          uniqueSignatures.add(`${firstName} ${lastName}`)
+        }
+      }
+    }
+
+    const signatures = Array.from(uniqueSignatures).join(', ');
+
+    return { signatures };
```

Comment:
Just `return signatures` here. We're getting rid of the nested object returns that Sails forced us into w/ this new API to be more RESTful in our responses.